apps/angular/src/core/mappers/anime.mapper.ts

```diff
@@ -0,0 +1,99 @@
+import { IMapper } from '@js-camp/core/mappers/imapper';
+
+import { Injectable } from '@angular/core';
+
+import { Anime } from '../models/anime';
+import { AnimeDto } from '../dtos/anime.dto';
+import { AnimeStatusDto } from '../dtos/anime-status.dto';
+import { AnimeStatus } from '../models/anime-status';
+import { AnimeTypeDto } from '../dtos/anime-type.dto';
+import { AnimeType } from '../models/anime-type';
+
+/** Map AnimeStatusDto to AnimeStatus. */
+const statusMappingFromDto: Record<AnimeStatusDto, AnimeStatus> = {
```

Comment:
You should use capitalized snake case for constants. Also dont forget about `readonly`.
```suggestion
const STATUS_FROM_DTO: Readonly<Record<AnimeStatusDto, AnimeStatus>> = {
```